lib/src/stepper.dart

```diff
@@ -20,28 +20,37 @@ class MultiStepperView extends StatelessWidget {
     this.currentStep = 0,
     this.showAllSteps = false,
     this.zeroIndexed = false,
-    this.showAllStepExpandedOne = false,
+    this.showOnlyCurrentStep = false,
     this.onStepTapped,
     this.paddingLeft = 10,
     this.paddingRight = 20,
     this.pageHeight = 400,
     this.theme = const StepperTheme(),
     Key? key,
-  }) : super(key: key);
+  })  : assert(
+          !(showOnlyCurrentStep && !showAllSteps),
+          'showOnlyCurrentStep only has effect if showAllSteps is true',
+        ),
+        super(key: key);
 
   /// The steps of the stepper.
   final List<MultiViewStep> steps;
 
   /// The index of the current step. If [steps] is not provided, [currentStep]
   final int currentStep;
 
-  /// Whether two show all the [steps] or just the current one.
+  /// Whether two show all the [steps] with indicators or just the current step.
+  ///
+  /// If you want to show all indicators but only 1 step at a time, set this to
+  /// true and set [showOnlyCurrentStep] to true.
   final bool showAllSteps;
 
   /// Whether to show 0 as the first step.
+  ///
+  /// VISUAL INDICATOR ONLY!
   final bool zeroIndexed;
 
-  // int callback for step clicked
+  /// int callback for step clicked
```

Comment:
If you're editing the line anyway, it could be a bit more descriptive. Something like `Callback which is called when a step is clicked`. We have no character limit here so no need to be so short :wink: